test/jit/test_torchbind.py

```diff
@@ -130,6 +130,47 @@ def foo():
         self.assertEqual(out[1].pop(), "mom")
         self.assertEqual(out[1].pop(), "hi")
 
+    def test_torchbind_def_property(self):
+        def foo_getter_setter_full():
```

Comment:
Nit:

Consider splitting `test_torchbind_def_property` into three test cases, so that they can be better parallelized in testing.

Reply:
yep will do on the final commit.